src/main/java/by/itech/lab/supplier/repository/RetailerRepository.java

```diff
@@ -0,0 +1,31 @@
+package by.itech.lab.supplier.repository;
+
+import by.itech.lab.supplier.domain.Customer;
+import by.itech.lab.supplier.domain.Item;
+import by.itech.lab.supplier.domain.Retailer;
+import by.itech.lab.supplier.domain.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface RetailerRepository extends JpaRepository<Retailer, Long> {
+    Optional<Retailer> findById(Long id);
```

Comment:
There is the same method in CrudRepository interface.
https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html#findById-ID-